spotty/providers/aws/config/validation.py

```diff
@@ -30,6 +31,8 @@ def validate_instance_parameters(params: dict):
                                              ),
         Optional('managedPolicyArns', default=[]): [str],
         Optional('instanceProfileArn', default=None): str,
+        Optional('inboundIp', default='0.0.0.0/0'): Use(IPNetwork, error='"inboundIp" should be a valid IP '
```

Comment:
For the default value, the `inbound_ip.version == 6` check will fail later in the code as the `Use(IPNetwork, ...` function is not applied. It can be fixed with the `IPNetwork('0.0.0.0/0')` value, but as I mentioned in another comment, we need to use `None` instead to know when the value wasn't set at all.